bentoml/_internal/frameworks/fastai.py

```diff
@@ -0,0 +1,306 @@
+    if model.info.module not in (MODULE_NAME, __name__):
+        raise NotFound(
+            f"Model {model.tag} was saved with module {model.info.module}, failed loading with {MODULE_NAME}."
+        )
+    return model
+
+
+def load_model(
+    bento_model: str | Tag | bentoml.Model,
+    cpu: bool = True,
+    pickle_module: types.ModuleType = cloudpickle,
+) -> Learner:
+    """
+    Load the :code:`fastai.learner.Learner` model instance with the given tag from the local BentoML model store.
+
+    Args:
+        bento_model (``str`` ``|`` :obj:`~bentoml.Tag` ``|`` :obj:`~bentoml.Model`):
+            Either the tag of the model to get from the store, or a BentoML `~bentoml.Model`
+            instance to load the model from.
+        cpu (``bool``, `optional`, default to ``True``):
+            Whether to load the model to CPU. if ``True``, the ``nn.Module`` is mapped to ``cpu`` via :code:`map_location` in ``torch.load``.
+            The loadded dataloader will also be converted to ``cpu``.
+
+            .. admonitions:: About :code:`cpu=True`
+
+                If model uses **mixed_precision**, then the results models will also be converted to FP32. Learn more about `mixed precision <https://docs.fast.ai/callback.fp16.html>`.
+        pickle_module (``types.ModuleType``, `optional`, default to ``cloudpickle``):
+            The pickle module to use for loading the model. This is passthrough to ``torch.load``.
+
+    Returns:
+        :code:`fastai.learner.Learner`:
+            The :code:`fastai.learner.Learner` model instance loaded from the model store or BentoML :obj:`~bentoml.Model`.
+
+    Example:
+    .. code-block:: python
+
+        import bentoml
+
+        model = bentoml.fastai.load_model("fai_learner")
+        results = model.predict({"input": "some input"})
+    """  # noqa
+    if not isinstance(bento_model, bentoml.Model):
+        bento_model = get(bento_model)
+
+    if bento_model.info.module not in (MODULE_NAME, __name__):
+        raise BentoMLException(
+            f"Model {bento_model.tag} was saved with module {bento_model.info.module}, failed loading with {MODULE_NAME}."
+        )
+
+    return t.cast(
+        Learner,
+        load_learner(
+            bento_model.path_of(MODEL_FILENAME), cpu=cpu, pickle_module=pickle_module
+        ),
+    )
+
+
+def save_model(
+    name: str,
+    learner: Learner,
+    *,
+    pickle_module: types.ModuleType = cloudpickle,
+    signatures: ModelSignaturesType | None = None,
+    labels: dict[str, str] | None = None,
+    custom_objects: dict[str, t.Any] | None = None,
+    metadata: dict[str, t.Any] | None = None,
+) -> Tag:
+    """
+    Save a :code:`fastai.learner.Learner` model instance to the BentoML model store.
+
+    Args:
+        name (``str``):
+            The name to give to the model in the BentoML store. This must be a valid
+            :obj:`~bentoml.Tag` name.
+        model (:code:`fastai.learner.Learner`):
```

Comment:
We should try to use `obj` for external types too; eventually we might be able to link to the relevant documentation.

```suggestion
        model (:obj:`fastai.learner.Learner`):
```